config.go

```diff
@@ -157,9 +157,10 @@ type Config struct {
 	// we can become a leader of a cluster containing only this node.
 	ShutdownOnRemove bool
 
-	// TrailingLogs controls how many logs we leave after a snapshot. This is
-	// used so that we can quickly replay logs on a follower instead of being
-	// forced to send an entire snapshot.
+	// TrailingLogs controls how many logs we leave after a snapshot. This is used
+	// so that we can quickly replay logs on a follower instead of being forced to
+	// send an entire snapshot. The value passed here is the initial setting used.
+	// This can be tuned during operation using
```

Comment:
TODO:

- [ ]  fix this comment and docs generally if we are happy with this API approach (duplicating fields)

Reply:
```suggestion
	// This can be tuned during operation using ReloadConfig.
```